MachUpskillingFY17/JabbR-Core
# Heather/commands - AddAdmin, AddOwner, Nudge

I put several of my commands into one branch. I implemented /add and remove admin, add and remove owners, list owners, and nudge.

Having difficulty testing nudge but I am getting a notification that the user was nudged.

To test:

1.  Log in and create a new room.

2.  Add/Remove admin

- in a chat room type /addadmin <username> and you should get a notification that the admin was added
-/removeadmin <username> will tell you that they are no longer an admin

3. Add/Remove owner

- in a chatroom type /addowner <username> and you will see that the owner is added. The ui sometimes updates the first time but sometimes takes a few tries of toggling between the rooms to update.
- if you type /owners you will get a list of the owners for the room
- Go to the account page and you should see a list of the rooms that user owns

4. Nudge user
- type /nudge <username> and you will get a notification that the user was nudged.
- If you try again within 60 seconds you will get an error that you can't do that

5. nudge room 
- type /nudge and the whole room will be nudged.
